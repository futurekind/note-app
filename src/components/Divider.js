import React from 'react';
import styled from 'styled-components';

import { colors } from '../utils/styles'

const View = styled.hr`
    height: 1px;
    margin: 0;
    border: none;
    border-bottom: 1px solid #fff;
    background: ${props => props.bg};
`

const Divider = props => <View bg={ colors.darkAlpha } />

export default Divider