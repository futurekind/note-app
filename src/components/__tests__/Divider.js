import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Divider from '../Divider';

describe('Divider', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(
            <Divider />
        )
    })

    it('renders', () => {
        expect(toJSON(wrapper)).toMatchSnapshot()
    })
})