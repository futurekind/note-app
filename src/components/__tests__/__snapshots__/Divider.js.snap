exports[`Divider renders 1`] = `
<styled.hr
  bg="rgba(53, 53, 53, 0.18)" />
`;
